back-end/src/main/java/com/PhanLam/backend/model/Course.java

```diff
@@ -86,14 +86,14 @@
     private CourseLevel courseLevel;
     
     // @Max(value=?)  @Min(value=?)//if you know range of your decimal fields consider using these annotations to enforce field validation
-    @Basic (optional = false)
+    @Basic(optional = false)
     @NotNull
-    @Column (name = "TuitionFee", nullable = false, precision = 19, scale = 4)
+    @Column(name = "TuitionFee", nullable = false, precision = 19, scale = 4)
     private BigDecimal tuitionFee;
-    @Basic (optional = false)
+    @Basic(optional = false)
     @NotNull
-    @Column (name = "DateCreated", nullable = false)
-    @Temporal (TemporalType.TIMESTAMP)
+    @Column(name = "DateCreated", nullable = false)
+    @Temporal(TemporalType.TIMESTAMP)
```

Comment:
@BuiXuanNhanTriSE05211 File này chắc git bị lỗi index. Ông xóa hết đi rồi copy toàn bộ nội dung file này của tôi trong nhánh develop paste vào nhé.

Reply:
Done